docs/guides/measure-qubits.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Measure qubits
+description: Learn how to measure qubits, including constraints on where measurements can be used.
+---
+
+# Measure qubits
+
+To get information about a qubit's state, you have to _measure_ it onto a [classical bit](/api/qiskit/circuit#qiskit.circuit.Clbit).  This is an irreversible operation, and usually destroys entanglement and phase coherence between the target qubit and the rest of the system. Thus, after a qubit is measured, you can't perform any more instructions on that qubit, and each qubit can only be measured once. Additionally, there must be at least one classical register to store the measurement information.
+
+## Apply a measurement to a circuit
+
+There are several ways to apply measurement to a circuit:
+
+### `Measure` class
+
+The Qiskit [Measure](/api/qiskit/circuit#qiskit.circuit.Measure) class measures the specified qubits.
```

Comment:
How about renaming this to use the `QuantumCircuit.measure` method instead of the `Measure` class? That's what the code snippets use and usually the main entry point 🙂 You could also add a `Measure` class part below which explicitly uses it, i.e.
```
from qiskit.circuit import Measure

....
qc.append(Measure(), [0], [0])  # measure qubit 0 into clbit 0
```